src/couch_epi/src/couch_epi_sup.erl

```diff
@@ -145,6 +145,7 @@ merge([{Id, _, _, _, _, _} = Spec | Rest], Children) ->
 %% BEGIN couch_epi_plugin behaviour callbacks
 
 -compile([export_all]).
+-compile(nowarn_export_all).
```

Comment:
Please explicitly export following functions:
```
-export([
   app/0,
   providers/0,
   services/0,
   data_providers/0,
   data_subscriptions/0,
   processes/0,
   notify/3
]).
```

Reply:
After consulting with @iilyak, I moved the test in couch_epi_sup to a new file couch_epi_basic_test , changing it as necessary to pass, and exporting `plugin_childspecs/3` from couch_epi_sup for testing.